includes/data-stores/class-wcs-subscription-data-store-cpt.php

```diff
@@ -549,4 +549,27 @@ public function get_subscription_customer_ids() {
 			WHERE subscription_meta.meta_key = '_customer_user' AND posts.post_type = 'shop_subscription'"
 		);
 	}
+
+	/**
+	 * Deletes all rows in the postmeta table with the given meta key.
+	 *
+	 * @param string $meta_key The meta key to delete.
+	 */
+	public function delete_all_metadata_by_key( $meta_key ) {
+		// Set variables to workaround ambiguous parameters of delete_metadata()
```

Comment:
**Nitpick:** This comment looks like copypasta from the original code as we don't appear to be setting variables here.